Makefile

```diff
@@ -4,11 +4,47 @@ SHELL := /bin/bash
 MAKEFILE_DIR = $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 PYTHON_VERSION := 3.6
 PYTHONHASHSEED := "random"
+
+# docker specific
 COMPOSE_FILE := "$(MAKEFILE_DIR)/ci/docker-compose.yml"
-DOCKER := docker-compose -f $(COMPOSE_FILE)
-DOCKER_RUN := PYTHON_VERSION=${PYTHON_VERSION} $(DOCKER) run --rm
-PYTEST_OPTIONS :=
+DOCKER := PYTHON_VERSION=$(PYTHON_VERSION) docker-compose -f $(COMPOSE_FILE)
+DOCKER_UP := $(DOCKER) up --remove-orphans -d --no-build
+DOCKER_RUN := $(DOCKER) run --rm
+DOCKER_BUILD := $(DOCKER) build
+DOCKER_STOP := $(DOCKER) rm --force --stop
+
+DOCKER_RUN_COMMAND := echo "you should do 'make docker_run DOCKER_RUN_COMMAND=[you command]'"
+
+BACKENDS := clickhouse impala kudu-master kudu-tserver mysql omniscidb parquet postgres sqlite
```

Comment:
can you add some comments here (and elsewhere if needed), and ideally doc this in the development docs, that when a new backend is added this needs to be updated.